### PR TITLE
use deprecated badge instead and update the description

### PR DESCRIPTION
**Pull request by yin1999**
### Description

use deprecated badge instead and update the description

### CODE REVIEW

File: files/en-us/web/javascript/guide/functions/index.md [hunk 1]
```diff
@@ -721,9 +721,9 @@ JavaScript has several top-level, built-in functions:
...
-  - : The deprecated **`escape()`** method computes a new string in which certain characters have been replaced by a hexadecimal escape sequence. Use {{jsxref("Global_Objects/encodeURI", "encodeURI")}} or {{jsxref("Global_Objects/encodeURIComponent", "encodeURIComponent")}} instead.
-- {{jsxref("Global_Objects/unescape", "unescape()")}}
-  - : The deprecated **`unescape()`** method computes a new string in which hexadecimal escape sequences are replaced with the character that it represents. The escape sequences might be introduced by a function like {{jsxref("Global_Objects/escape", "escape")}}. Because `unescape()` is deprecated, use {{jsxref("Global_Objects/decodeURI", "decodeURI()")}} or {{jsxref("Global_Objects/decodeURIComponent", "decodeURIComponent")}} instead.
+- {{jsxref("Global_Objects/escape", "escape()")}} {{deprecated_inline}}
+  - : The **`escape()`** method computes a new string in which certain characters have been replaced by a hexadecimal escape sequence. It's deprecated and non-standard, please use {{jsxref("Global_Objects/encodeURI", "encodeURI()")}} or {{jsxref("Global_Objects/encodeURIComponent", "encodeURIComponent()")}} instead.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/functions/index.md:725]:
```suggestion
  - : The **`escape()`** method computes a new string in which certain characters have been replaced by a hexadecimal escape sequence. It's deprecated and you should use {{jsxref("Global_Objects/encodeURI", "encodeURI()")}} or {{jsxref("Global_Objects/encodeURIComponent", "encodeURIComponent()")}} instead.
```

File: files/en-us/web/javascript/guide/functions/index.md [hunk 2]
```diff
@@ -721,9 +721,9 @@ JavaScript has several top-level, built-in functions:
...
+- {{jsxref("Global_Objects/unescape", "unescape()")}} {{deprecated_inline}}
+  - : The **`unescape()`** method computes a new string in which hexadecimal escape sequences are replaced with the character that it represents. The escape sequences might be introduced by a function like {{jsxref("Global_Objects/escape", "escape()")}}. It's deprecated and non-standard, please use {{jsxref("Global_Objects/decodeURI", "decodeURI()")}} or {{jsxref("Global_Objects/decodeURIComponent", "decodeURIComponent()")}} instead.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/functions/index.md:727]:
```suggestion
  - : The **`unescape()`** method computes a new string in which hexadecimal escape sequences are replaced with the character that it represents. The escape sequences might be introduced by a function like {{jsxref("Global_Objects/escape", "escape()")}}. It's deprecated and you should use {{jsxref("Global_Objects/decodeURI", "decodeURI()")}} or {{jsxref("Global_Objects/decodeURIComponent", "decodeURIComponent()")}} instead.
```

**Comment by yin1999** [comment]
Deprecated is what matters here. Thanks for the suggestions @Josh-Cena :)

File: files/en-us/web/javascript/guide/functions/index.md [hunk 3]
```diff
@@ -721,9 +721,9 @@ JavaScript has several top-level, built-in functions:
...
-- {{jsxref("Global_Objects/escape", "escape()")}}
-  - : The deprecated **`escape()`** method computes a new string in which certain characters have been replaced by a hexadecimal escape sequence. Use {{jsxref("Global_Objects/encodeURI", "encodeURI")}} or {{jsxref("Global_Objects/encodeURIComponent", "encodeURIComponent")}} instead.
-- {{jsxref("Global_Objects/unescape", "unescape()")}}
-  - : The deprecated **`unescape()`** method computes a new string in which hexadecimal escape sequences are replaced with the character that it represents. The escape sequences might be introduced by a function like {{jsxref("Global_Objects/escape", "escape")}}. Because `unescape()` is deprecated, use {{jsxref("Global_Objects/decodeURI", "decodeURI()")}} or {{jsxref("Global_Objects/decodeURIComponent", "decodeURIComponent")}} instead.
+- {{jsxref("Global_Objects/escape", "escape()")}} {{deprecated_inline}}
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/functions/index.md:724]:
```suggestion
- {{jsxref("Global_Objects/escape", "escape()")}} {{Deprecated_Inline}}
```

File: files/en-us/web/javascript/guide/functions/index.md [hunk 4]
```diff
@@ -721,9 +721,9 @@ JavaScript has several top-level, built-in functions:
...
+  - : The **`escape()`** method computes a new string in which certain characters have been replaced by a hexadecimal escape sequence. It's deprecated and you should use {{jsxref("Global_Objects/encodeURI", "encodeURI()")}} or {{jsxref("Global_Objects/encodeURIComponent", "encodeURIComponent()")}} instead.
+- {{jsxref("Global_Objects/unescape", "unescape()")}} {{deprecated_inline}}
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/guide/functions/index.md:726]:
```suggestion
- {{jsxref("Global_Objects/unescape", "unescape()")}} {{Deprecated_Inline}}
```

**Review by Josh-Cena — APPROVED** [review]
Thanks :)

**Pull request merged**